src/main/scala/com/github/catalystcode/fortis/spark/streaming/facebook/FacebookUtils.scala

```diff
@@ -10,23 +10,35 @@ import org.apache.spark.streaming.StreamingContext
 import org.apache.spark.streaming.dstream.ReceiverInputDStream
 
 object FacebookUtils {
-  def createPageStream(
+  def createPageStreams(
     ssc: StreamingContext,
     auth: FacebookAuth,
-    pageId: String,
+    pageIds: Set[String],
     fields: Set[String] = Set("message", "place", "caption", "from", "name", "comments"),
     pollingSchedule: PollingSchedule = PollingSchedule(30, TimeUnit.SECONDS),
     pollingWorkers: Int = 1,
     storageLevel: StorageLevel = StorageLevel.MEMORY_ONLY
   ): ReceiverInputDStream[FacebookPost] = {
     new FacebookInputDStream(
       ssc = ssc,
-      client = new FacebookPageClient(
+      clients = pageIds.map(pageId => new FacebookPageClient(
         pageId = pageId,
         auth = auth,
-        fields = fields),
+        fields = fields)),
       pollingSchedule = pollingSchedule,
       pollingWorkers = pollingWorkers,
       storageLevel = storageLevel)
   }
+
+  def createPageStream(
+    ssc: StreamingContext,
+    auth: FacebookAuth,
+    pageId: String,
+    fields: Set[String] = Set("message", "place", "caption", "from", "name", "comments"),
+    pollingSchedule: PollingSchedule = PollingSchedule(30, TimeUnit.SECONDS),
+    pollingWorkers: Int = 1,
+    storageLevel: StorageLevel = StorageLevel.MEMORY_ONLY
+  ): ReceiverInputDStream[FacebookPost] = {
+    createPageStreams(ssc, auth, Set(pageId), fields, pollingSchedule, pollingWorkers, storageLevel)
+  }
 }
```

Comment:
we're going to need to expose a `createPageCommentsStream` utility to focus on newly posted comments since the previous processed time. This stream should crawl facebook4j `getPostComments` http://facebook4j.github.io/javadoc/facebook4j/api/PostMethods.html#getPostComments(java.lang.String, facebook4j.Reading)